### PR TITLE
improve build scripts: combine test.sh and build.sh into one

### PR DESCRIPTION
**Pull request by qinzuoyan**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -26,21 +26,28 @@ function usage_build()
 {
     echo "Options for subcommand 'build':"
     echo "   -h|--help         print the help info"
-    echo "   -d|--debug        build in debug mode, default is release"
+    echo "   -t|--type         build type: debug|release, default is debug"
```
Comment (imzhenyu) [inline run.sh:29]: what about relwithdebinfo? Which is useful for debugging when bad things happen on bed?

Comment (qinzuoyan) [inline run.sh:29]: relwithdebinfo? what's this?

Comment (imzhenyu) [inline run.sh:29]: There are two other build types besides debug and release...

Sent from my Windows Phone

---

发件人: Zuoyan Qinmailto:notifications@github.com
发送时间: ‎2015/‎12/‎15 16:11
收件人: imzhenyu/rDSNmailto:rDSN@noreply.github.com
抄送: Zhenyu Guomailto:imzhenyu@outlook.com
主题: Re: [rDSN] improve build scripts: combine test.sh and build.sh into one (#88)

> @@ -26,21 +26,28 @@ function usage_build()
>  {
>      echo "Options for subcommand 'build':"
>      echo "   -h|--help         print the help info"
> -    echo "   -d|--debug        build in debug mode, default is release"
> -    echo "   -t|--type         build type: debug|release, default is debug"

relwithdebinfo? what's this?

---

Reply to this email directly or view it on GitHub:
https://github.com/imzhenyu/rDSN/pull/88/files#r47607251

Comment (qinzuoyan) [inline run.sh:29]: I am not very clear about them, maybe we can add them when needed in the future.

**Pull request merged**